modules/cli/src/main/scala/scala/cli/commands/shared/ScalaJsOptions.scala

```diff
@@ -59,6 +59,11 @@ final case class ScalaJsOptions(
   @HelpMessage("Enable jsdom")
     jsDom: Option[Boolean] = None,
 
+  @Group(HelpGroup.ScalaJs.toString)
+  @Tag(tags.experimental)
+  @HelpMessage("Experimental: emit WASM")
```

Comment:
It's sufficient that it's tagged as `experimental`.
```suggestion
  @HelpMessage("Emit WASM")
```